tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
     "esModuleInterop": true,
-    "lib": ["es2015", "es2016", "esnext"]
+    "lib": ["es2015", "es2016", "esnext", "DOM"]
```

Comment:
I have no idea if this is important or not, it probably is not, but they have it as lower-case `dom` ?
https://github.com/firebase/firebase-js-sdk/blob/49ee786f2b022e65aef45693e1a8b546d889ec10/packages/messaging/tsconfig.json#L7